src/main/docs/guide/azureCosmos.adoc

```diff
@@ -0,0 +1,9 @@
+Micronaut Data Azure Cosmos supports some of the things that are possible to do with JPA and JDBC/R2DBC implementations, including:
+
+- <<azureCosmosRepositories, Repositories compile-time generated>> and <<projections, projection>> queries
+- <<azureCosmosAttributeConverter, Attribute converters>>
+- <<azureCosmosOptimisticLocking, Optimistic locking>>
+
+Cascading and joins are not supported like in the rest of data modules. More about <<azureCosmosSpecifics, specifics>> can be seen here.
+
+The interaction between the object layer and Azure Cosmos Db serialization/deserialization is implemented using https://micronaut-projects.github.io/micronaut-serialization/1.0.x/guide[Micronaut Serialization].
```

Comment:
here you add `Db` at the end. We should be consistent and refer to it as just `Azure Cosmos` or `Azure Cosmos DB` all the time

Reply:
Updated trying to make it consistent